index.js

```diff
@@ -0,0 +1,69 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+let button = document.querySelector("button");
+button.addEventListener("click", addReview);
+
+})
+
+let url = "https://ghibliapi.herokuapp.com/films";
+
+
+axios.get(url)
```

Comment:
We shouldn't fire a network request before knowing that the DOM is loaded, specially if once we get the response from the request we are touching the DOM with stuff like `getElementById`. Even though we are almost always guarantied that the network request would take more time than loading the DOM, we shouldn't rely on it. What if we have a website jam packed with elements and the network request wins the race against the DOM loading, your website would break. How would you approach this. A hint: Wrap this axios call in a function. When would you call it though? Also maybe create another function that handles the displaying of the info (creating and appending the option)